pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -63,11 +63,11 @@ type BackingServiceSelector struct {
 // +k8s:openapi-gen=true
 type ApplicationSelector struct {
 	// +optional
-	MatchLabels map[string]string `json:"matchLabels,omitempty"`
-	Group       string            `json:"group,omitempty"`
-	Version     string            `json:"version"`
-	Resource    string            `json:"resource"`
-	ResourceRef string            `json:"resourceRef"`
+	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
+	Group         string                `json:"group,omitempty"`
```

Comment:
Perhaps GroupVersionResource can be embedded here as well?

Reply:
We can do that in a separate PR.